src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int hypotenuse;
+    private int altitude;
+    private int base;
+
+    public RightTriangle(String color, int base, int altitude) {
+        super(color);
+        this.altitude = altitude;
+        this.base = base;
+        this.hypotenuse = (int)Math.sqrt(base * base + altitude * altitude);
```

Comment:
it's a bad practice to add such logic in the constructor, you should pass to the constructor already formed parameters, so let's remove field hypotenuse at all